Tests/RegressionTests.cs

```diff
@@ -229,22 +229,22 @@ private static TestCaseData[] GetRegressionTestParameters()
                 {"Total Trades", "67"},
                 {"Average Win", "1.13%"},
                 {"Average Loss", "-0.69%"},
-                {"Compounding Annual Return", "17.718%"},
+                {"Compounding Annual Return", "17.739%"},
                 {"Drawdown", "5.100%"},
-                {"Expectancy", "0.813"},
-                {"Net Profit", "17.718%"},
-                {"Sharpe Ratio", "1.38"},
+                {"Expectancy", "0.814"},
+                {"Net Profit", "17.739%"},
+                {"Sharpe Ratio", "1.381"},
                 {"Loss Rate", "31%"},
                 {"Win Rate", "69%"},
                 {"Profit-Loss Ratio", "1.64"},
                 {"Alpha", "0.055"},
                 {"Beta", "0.379"},
                 {"Annual Standard Deviation", "0.099"},
                 {"Annual Variance", "0.01"},
-                {"Information Ratio", "-0.703"},
+                {"Information Ratio", "-0.702"},
```

Comment:
I'm unsure of why there are changes here since the algorithm doesn't use a security that has decimalized quantity. Did we inspect why these values change? Can we define unit tests that would have failed *before* this change and that now pass due to his change (and subsequently describes why these values have changed).

I'm always hesitant when a regression test changes and I can't definitely say why.